components/sftp/sftp.app.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
+  type: "app",
+  app: "sftp",
+  props: {
+    db: "$.service.db"
+  },
+  methods: {
+    config: {
```

Comment:
Do you need access to the `config` prop in other components or is it purely used within the `getClient` method of the app file?

If the latter, I'd recommend removing it - I'll clarify in comments in the `getClient` method below.